lib/src/browser.dart

```diff
@@ -57,20 +61,23 @@ class Browser {
     _firefox,
     _safari,
     _internetExplorer,
+    _edge,
     _wkWebView
   ];
 
   bool get isChrome => this == _chrome;
   bool get isFirefox => this == _firefox;
   bool get isSafari => this == _safari;
   bool get isInternetExplorer => this == _internetExplorer;
+  bool get isEdge => this == _edge;
```

Comment:
my vote is with @aaronlademann-wf so unless you strongly object, I would say change it:
```dart
bool get isInternetExplorer => this == _internetExplorer || isEdge;
bool get isEdge => this == _edge;
```
in Canary you can switch based on the isEdge bool